### PR TITLE
Commodore? Where?

### PR DESCRIPTION
**Pull request by realforest2001**
## About The Pull Request

Replaces most references to commodore with colonel

## Why It's Good For The Game
Continuity/Immersion. Council keep walking around wearing commodore berets.

## Changelog

:cl:
fix: majority of remaining IC references to commodores have been thoroughly nuked.
tweak: CO Senator set to MO6 full Colonel with permission from lore team.
/:cl:

### CODE REVIEW

**Review by nauticall — APPROVED** [review]
just basic find and replace, should be good

File: code/modules/clothing/head/head.dm [hunk 1]
```diff
@@ -413,23 +413,23 @@
...
+	name = "marine colonel beret"
+	desc = "An armored blue beret with the Lieutenant Colonel's insignia emblazoned on it. Its blue color symbolizes loyalty, confidence, and politics - the core components of a true Colonel."
 	icon_state = "cdreberet"
 
-/obj/item/clothing/head/beret/marine/commander/cdrechief
-	name = "marine commodore-in-chief beret"
-	desc = "A dark blue, custom-tailored armored beret signifying the Chief of all Commodores. Definitely not an alias for an General."
+/obj/item/clothing/head/beret/marine/commander/councilchief
+	name = "marine colonel beret"
+	desc = "A dark blue, custom-tailored armored beret signifying The Colonel. Definitely not an alias for an General."
```
Comment (stanalbatross) [inline code/modules/clothing/head/head.dm:432]:
```suggestion
	desc = "A dark blue, custom-tailored armored beret signifying the Colonel. Definitely not an alias for an General."
```

File: maps/templates/Chinook.dmm [hunk 2]
```diff
@@ -1232,7 +1221,7 @@
 "dZ" = (
 /obj/structure/machinery/door/airlock/almayer/command{
 	dir = 1;
-	name = "Commodore, Fifth Division"
+	name = "Colonel, Fifth Division"
```
Comment (stanalbatross) [inline maps/templates/Chinook.dmm:1224]:
```suggestion
	name = "\improper Colonel, Fifth Division"
```

File: maps/templates/Chinook.dmm [hunk 3]
```diff
@@ -2852,7 +2841,7 @@
 "jg" = (
 /obj/structure/machinery/door/airlock/almayer/command{
 	dir = 1;
-	name = "Commodore, First Division"
+	name = "Colonel, First Division"
```
Comment (stanalbatross) [inline maps/templates/Chinook.dmm:2844]:
```suggestion
	name = "\improper Colonel, First Division"
```

File: maps/templates/Chinook.dmm [hunk 4]
```diff
@@ -3173,7 +3162,7 @@
 "ki" = (
 /obj/structure/machinery/door/airlock/almayer/command{
 	dir = 1;
-	name = "Commodore, Second Division"
+	name = "Colonel, Second Division"
```
Comment (stanalbatross) [inline maps/templates/Chinook.dmm:3165]:
```suggestion
	name = "\improper Colonel, Second Division"
```

File: maps/templates/Chinook.dmm [hunk 5]
```diff
@@ -5610,7 +5592,7 @@
 "rX" = (
 /obj/structure/machinery/door/airlock/almayer/command{
 	dir = 1;
-	name = "Commodore, Fourth Division"
+	name = "Colonel, Fourth Division"
```
Comment (stanalbatross) [inline maps/templates/Chinook.dmm:5595]:
```suggestion
	name = "\improper Colonel, Fourth Division"
```

**Review by stanalbatross — CHANGES_REQUESTED** [review]

File: code/modules/clothing/head/head.dm (hunk 1 above)
Comment (AnCopper) [inline code/modules/clothing/head/head.dm:432]:
```suggestion
	desc = "A dark blue, custom-tailored armored beret signifying The Colonel. Definitely not an alias for a General."
```

Comment (realforest2001) [reply code/modules/clothing/head/head.dm:432]: I can change if needs be, but this was a deliberate emphatic capitalisation.

**Comment by silencer-pl** [comment]
Just a quick note, something something great minds, I also added a Colonel gearset as an admin spawn in #1033 but it honestly seems like these two don't conflict with each other at all, since mine is meant for ERTs in is a separate job/gearset and not meant to be used in normal rounds.

**Review by stanalbatross — APPROVED** [review]

**Pull request merged**